source/_integrations/sighthound.markdown

```diff
@@ -63,3 +63,47 @@ source:
       required: false
       type: string
 {% endconfiguration %}
+
+To verify the integration, check if new entities appear as `camera.sighthound` and `image_processing.sighthound_camera_my_cam`
+
+## Process an Image
+
+When you want to process an image, you have to call `image_processing.scan` service and listen to the sighthound.person_detected and/or sighthound.vehicle_detected events.
```

Comment:
Calling `scan` is the same for all image processing integrations

Reply:
Not everybody is familiar with the image processing integrations. I can remove, if it bothers you.

Reply:
```suggestion
When you want to process an image, you have to call `image_processing.scan` service and listen to the `sighthound.person_detected` and/or `sighthound.vehicle_detected` events.
```